src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -206,6 +200,10 @@
   <RuntimeSymbolsPath>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</RuntimeSymbolsPath>
   <LLDBHelperScript>$(ScriptRootDir)/lldbhelper.py</LLDBHelperScript>
 
+  <BuildType>nightly</BuildType>
```

Comment:
You have the condition switched compared to the similar logic in the Windows config.